tests/Collection/CollectionFunctionalTest.php

```diff
@@ -29,9 +29,9 @@ public function testFindOne()
             'sort' => array('x' => -1),
         );
 
-        $expected = array('_id' => 3, 'x' => 33);
+        $expected = (object) array('_id' => 3, 'x' => 33);
```

Comment:
This is fine for now. I opened [PHPLIB-113](https://jira.mongodb.org/browse/PHPLIB-113) to track creation of a typeMap option for `findOne()`.